common4j/build.gradle

```diff
@@ -218,6 +218,9 @@ dependencies {
     testFixturesCompileOnly "org.projectlombok:lombok:$rootProject.ext.lombokVersion"
     testFixturesAnnotationProcessor "org.projectlombok:lombok:$rootProject.ext.lombokVersion"
     resolvableTestFixturesImplementation "org.robolectric:junit:$rootProject.ext.robolectricVersion"
+
+    implementation platform("io.opentelemetry:opentelemetry-bom:1.16.0")
```

Comment:
question : here we added the dependency in common4j and did not have to add it in build.gradle of common. But in your other broker PR.. you added the above dependency once in broker4j and in AADAuthenticator as well. Why is that so?

Reply:
We didn't add it in `common` because as of now we're not capturing any data that is in common code specifically. If we want to capture data in `common` code then we would need to add it to `common` as well. 

I added it to `broker4j` and `AADAuthenticator` because using Open Telemetry in both of these projects. If we don't want to explicitly add these in every project and rather prefer these to carry over from other projects i.e. carry over from broker4j to AADAuthenticator, then we need to enable `transitive` dependencies. These are currently off in AADAuthenticator:

```groovy
    distApi("com.microsoft.identity:broker4j:${broker4jVersion}") {
        transitive = false
    }
```